lua/nvim-treesitter/configs.lua

```diff
@@ -192,7 +208,7 @@ function M.setup(user_data)
       end
       if config.modules[mod].keymaps and type(data.keymaps) == 'table' then
         for f, map in pairs(data.keymaps) do
-          if config.modules[mod].keymaps[f] then
+          if config.modules[mod].keymaps[f] or mod == 'textobjects' then
```

Comment:
This module use the inverse of the keymap. Maybe we should remove the feature that users define textobjects directly in config.